README.md

```diff
@@ -222,14 +211,87 @@ import './styles.scss'
 }
 ```
 
-## Development
 
-- Clone this repository
-- Install dependencies using `yarn install` or `npm install`
-- Start development server using `yarn run dev` or `npm run dev`
-- Point your browser to `http://localhost:3000`
-- You will see a different colour based on the variant set for you
-- In order to test your luck, try clearing your cookies and see if the background colour changes or not
+Inject global styles into your layout.
+
+`layouts/default.vue`:
+
+```vue
+<template>
+  <nuxt/>
+</template>
+
+<script>
+export default {
+      head () {
+        return {
+            bodyAttrs: {
+                class: this.$exp.$classes.join(' ')
+            }
+        }
+    },
+}
+</script>
+```
+
+This would render:
+
+```html
+<body class="exp-background-color-2">
+```
+
+## Integrating with Google Tag Manager
+There are many ways to integrate Google Optimize with GTM.  Usually you have Google Analytics setup in GTM and you push the experiment token into GTM.
+
+### Method 1 - Push Token To the Data Layer
+
+This method pushes the exp value directly into the dataLayer to be used within a GTM tag.
+
+- In GTM, create a new variable called `exp`.
+
+- Next, create a new variable of type `Google Analytics Settings`.
+
+- Under `Fields to Set`, add a field with the name `exp` and value `{{ exp }}`
+
+- In your code, pass the token value to GTM
+```js
+// directly
+dataLayer.push({ exp: window.$nuxt.$exp.token })
+
+// or using the Nuxt GTM module
+this.$gtm.push({ exp: this.$exp.token })
+```
+
+Now when you trigger GA pageviews or events the exp value will be attached to the user.
+
+### Method 2 - Call Google Analytics Yourself From GTM
+
+- Create a tag in GTM of type Html
+- Use whatever trigger you like
+- Fill the tag with the following content
+```js
+<script>window.ga('set', 'exp', window.$nuxt.$exp.token)</script>
+```
+
+This will call the Google Analytics lib directly and report the experiment assignment for the current user whenever you activate the trigger.
+
+
+## emitOnLoad Caution
+
+By default emitOnLoad is true (for backward compatability).  This means that when a user hits your site this plugin will tell Google Analytics (Optimize) that the user is in the experiment. 
```

Comment:
I think it depends. Not all experiments are specific to some pages so `backward compatibility` is misleading here even if in next major we are going to disable by default. There could be also smarter ways like defining if an experiment is global or manual in experiment export file and skip auto emit if is not.